src/index.js

```diff
@@ -18,7 +18,8 @@ export default function (sails) {
             config[job].onComplete,
             typeof config[job].start === 'boolean' ? config[job].start : true,
             config[job].timezone,
-            config[job].context
+            config[job].context,
+            typeof config[job].runOnInit === 'boolean' ? config[job].runOnInit : true
```

Comment:
Would be better to set `false` by default, just to make sure that users don't face unexpected behavior when restarting server. I mean, some tasks can be some highly CPU consuming tasks and that's not ok to run them every time server is starting. What you think?

Reply:
Sound reasonable :+1: